app/models/user.rb

```diff
@@ -85,7 +85,7 @@ def self.new_token
   end
 
   def admin
-    return true if self.email == "lhsdvm@aol.com"  || self.email == "testuser@berkeley.edu"
+    return true if self.email == "lhsdvm@aol.com"  || self.email == "testuser@berkeley.edu"  || self.email == "xshuyin@berkeley.edu"
```

Comment:
Should this still be here or was it for testing?

Reply:
that's my bad thanks !